packages/cmf/src/onEvent.js

```diff
@@ -0,0 +1,79 @@
+import get from 'lodash/get';
+
+function serializeEvent(event) {
+	if (event.persist) {
+		event.persist();
+	}
+	return event;
+}
+
+function getOnEventActionCreatorHandler(instance, config, original) {
```

Comment:
why don't you name the third argument like in the 2 other functions : currentHandler? 
It has semantic which is better for comprehension than old/original/previous.

Reply:
+1 please rename this one